recipe/meta.yaml

```diff
@@ -47,8 +47,7 @@ about:
     plotting, summary and reporting of `mefa' objects. Reports can be generated in plain
     text or LaTeX format. Vignette contains worked examples.
   license_family: GPL2
-  license_file: '{{ environ["PREFIX"] }}\/lib\/R\/share\/licenses\/GPL-2'  # [unix]
```

Comment:
Looks like the bot missed this line because of the backslashes in front of the forward slashes. Hopefully there aren't too many recipes that deviate from the common pattern.

Reply:
I have been fixing a few of them already.

Reply:
OK. Then I'll add another regex to catch this edge case.

Reply:
xref: https://github.com/regro/cf-scripts/pull/413

Reply:
@bgruening This should be fixed now. Please let me know if you continue to have this or other problems with the automated migration PRs.

Reply:
Cool, thanks. I hope the migration is largely done. But I will keep an eye open.